Cargo.lock

```diff
@@ -5060,7 +5060,7 @@ dependencies = [
 [[package]]
 name = "ruma"
 version = "0.10.1"
-source = "git+https://github.com/matrix-org/ruma?rev=f25b3220d0c3ece7720020ed180af4955a855402#f25b3220d0c3ece7720020ed180af4955a855402"
+source = "git+https://github.com/toger5/ruma?rev=195cc6f57464b541d6ec0e8368c0993b3f62a760#195cc6f57464b541d6ec0e8368c0993b3f62a760"
```

Comment:
this needs to be changed back to matrix-org and the correct hash once the sss-feat branch is rebased onto the required ruma updates: https://github.com/ruma/ruma/pull/1883